llarp/link/link_manager.cpp

```diff
@@ -384,10 +389,19 @@ namespace llarp
   }
 
   void
-  LinkManager::gossip_rc(const RemoteRC& rc)
+  LinkManager::gossip_rc(const RouterID& rc_rid, std::string serialized_rc)
   {
     for (auto& [rid, conn] : ep.conns)
-      send_control_message(rid, "gossip_rc", std::string{rc.view()}, nullptr);
+    {
+      // don't send back to the owner...
+      if (rid == rc_rid)
+        continue;
+      // don't gossip RCs to clients
+      if (not conn->remote_is_relay)
+        continue;
+
+      send_control_message(rid, "gossip_rc", serialized_rc, nullptr);
```

Comment:
Minor:
Same thing about the default callback parameter

Reply:
Yup, just being explicit to convey intent.